### PR TITLE
(+semver: breaking) Updated testing/docker package

### PR DESCRIPTION
**Pull request by thzinc**
This is a proper bump to the module version. The new module path is `github.com/syncromatics/go-kit/v2` and will be tagged as such.

---

(+semver: breaking) Updated testing/docker package

- Unified names of functions across docker package
- Updated module to v2
- Added RabbitMQ support
- Added test coverage
- Added postgres port support

### CODE REVIEW

**Review by cuevaskoch — APPROVED** [review]

**Comment by annymsMthd** [comment]
Is this something in the go modules doc?

**Review by joscalv — APPROVED** [review]

**Review by dkaminski — APPROVED** [review]

**Comment by thzinc** [comment]
This has an issue. I might need to actually create or copy files into a `v2` folder to support this.

@annymsMthd, I found this after I published the PR: https://blog.golang.org/v2-go-modules. Go tooling is super pedantic about semver and requires breaking/major version changes to actually exist in _vN_ folders

**Comment by annymsMthd** [comment]
@thzinc ouch. So if you go to v2 it requires the folder even if you tag the repo v2?

**Comment by annymsMthd** [comment]
oh is see. this package did something similar https://github.com/vektah/gqlparser/compare/v2.0.0...master

**Review by annymsMthd — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
 	github.com/grpc-ecosystem/go-grpc-middleware v1.1.0
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
-	github.com/klauspost/cpuid v1.2.3 // indirect
 	github.com/opentracing/opentracing-go v1.1.0
 	github.com/phayes/freeport v0.0.0-20180830031419-95f893ade6f2
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v0.9.3-0.20190127221311-3c4408c8b829
 	github.com/rakyll/statik v0.1.6
-	github.com/samuel/go-zookeeper v0.0.0-20190923202752-2cc03de413da // indirect
+	github.com/syncromatics/go-kit v1.3.0
```
Comment (annymsMthd) [inline go.mod]: I wonder why it's bringing back in v1. was there a ref missed?

**Pull request merged**